torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -19,15 +19,16 @@
 #include "pybind11/pybind11.h"
 #include "pybind11/pytypes.h"
 #include "pybind11/stl_bind.h"
-#include "tensorflow/compiler/xla/pjrt/distributed/distributed.h"
-#include "tensorflow/compiler/xla/python/profiler/internal/traceme_wrapper.h"
-#include "tensorflow/compiler/xla/service/hlo_parser.h"
-#include "tensorflow/core/example/example.pb.h"
-#include "tensorflow/core/example/feature.pb.h"
-#include "tensorflow/python/profiler/internal/profiler_pywrap_impl.h"
-#include "tensorflow/tsl/platform/env.h"
-#include "tensorflow/tsl/profiler/lib/traceme.h"
+#include "xla/pjrt/distributed/distributed.h"
+#include "xla/python/profiler/internal/traceme_wrapper.h"
+#include "xla/service/hlo_parser.h"
+#include "third_party/xla_client/example.pb.h"
```

Comment:
why is this file moved to `xla_client`? is it for prototyping purposes?

CC @JackCaoG @ManfeiBai

Reply:
yes, it is for prototyping purposes,
but I created a `example.pb.h` in `third_party/xla_client` directly because want to only add new files in `third_party/xla_client`